lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -178,7 +182,8 @@ def instructor_dashboard_2(request, course_id):
         'disable_buttons': disable_buttons,
         'analytics_dashboard_message': analytics_dashboard_message,
         'certificate_white_list': certificate_white_list,
-        'certificate_exception_url': certificate_exception_url
+        'certificate_exception_url': certificate_exception_url,
```

Comment:
I think we should rename `certificate_exception_url` to `generate_certificate_exceptions_url`.